symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -0,0 +1,394 @@
+package com.symphony.bdk.core.service.user;
```

Comment:
Not sure that we should put this service in a dedicated package. We can keep it as part of `com.symphony.bdk.core.service`